test.js

```diff
@@ -44,33 +46,132 @@ describe('Merge', function() {
     });
   });
 
-  it('should override objects of multiple objects', function() {
+  it('should override objects of multiple objects', function () {
     var a = {
-      foo: 'a',
+      foo: 'a'
     };
     var b = {
       foo: 'b'
     };
     var c = {
-      foo: 'c',
+      foo: 'c'
     };
 
     assert.deepEqual(merge(a, b, c), c);
   });
 
-  it('should append arrays of multiple objects', function() {
+  it('should append arrays of multiple objects', function () {
     var a = {
-      foo: ['a'],
+      foo: ['a']
     };
     var b = {
-      foo: ['b'],
+      foo: ['b']
     };
     var c = {
-      foo: ['c'],
+      foo: ['c']
     };
 
     assert.deepEqual(merge(a, b, c), {
       foo: ['a', 'b', 'c']
     });
   });
+
+  it('should overwride loader string values', function () {
```

Comment:
Yeah. That's a good solution! Maybe this behavior should be mentioned at the README. Apart from that I'm fine with this as it keeps the API clean.